var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -57,6 +58,10 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("umpire", when="+raja")
     depends_on("umpire@6.0.0:", when="@1.1.0: +raja")
 
+    depends_on("raja@0.14.0", when="@1.5.0 +raja")
+    depends_on("umpire@6.0.0", when="@1.5.0 +raja")
+    depends_on("camp@0.2.3", when="@1.5.0 +raja")
```

Comment:
```suggestion
    depends_on("umpire", when="+raja")

    depends_on("raja@0.14.0", when="@1.1.0: +raja")    
    depends_on("umpire@6.0.0", when="@1.1.0: +raja")
    depends_on("camp@0.2.3", when="@1.1.0: +raja")
```

For some reason, I can't include lines 56/57 in this suggestion, but I think this concrete version requirement extends all the way back to 1.1.0

Reply:
Hopefully future versions of exago can work with latest versions of these packages - as I suspect ancoring of these versions will cause grief [when more raja usage from other xsdk pkgs is enabled]

>  For some reason, I can't include lines 56/57 in this suggestion

If you can post the full diff - I can apply it.

Reply:
` depends_on("raja@0.14.0:", when="@1.1.0: +raja")` this is the line that I couldn't include in my GitHub integrated suggestion. The idea in the end is to just move that line further down the file, and to remove the `:` specifying the version range in RAJA. Final line is already included in my diff, you just have to delete quoted line.

Reply:
pushed a fix - might want re-review